backend/composer/models.py

```diff
@@ -229,7 +228,7 @@ class AnatomicalEntity(models.Model):
     """Anatomical Entity"""
 
     name = models.CharField(max_length=200, db_index=True)
-    ontology_uri = models.URLField(unique=True)
+    ontology_uri = models.URLField()
```

Comment:
This can't be unique because we can have children of anatomical entities with the same ontology uri